XamlControlsGallery/ControlPages/GridViewPage.xaml.cs

```diff
@@ -9,94 +9,504 @@
             {
-                Control1.FlowDirection = FlowDirection.RightToLeft;
+                ContentGridView.FlowDirection = FlowDirection.RightToLeft;
             }
             else
             {
-                Control1.FlowDirection = FlowDirection.LeftToRight;
+                ContentGridView.FlowDirection = FlowDirection.LeftToRight;
             }
         }
 
         private void SelectionModeComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            if (Control1 != null)
+            if (ContentGridView != null)
             {
                 string colorName = e.AddedItems[0].ToString();
                 switch (colorName)
                 {
                     case "None":
-                        Control1.SelectionMode = ListViewSelectionMode.None;
+                        ContentGridView.SelectionMode = ListViewSelectionMode.None;
                         SelectionOutput.Text = string.Empty;
                         break;
                     case "Single":
-                        Control1.SelectionMode = ListViewSelectionMode.Single;
+                        ContentGridView.SelectionMode = ListViewSelectionMode.Single;
                         break;
                     case "Multiple":
-                        Control1.SelectionMode = ListViewSelectionMode.Multiple;
+                        ContentGridView.SelectionMode = ListViewSelectionMode.Multiple;
                         break;
                     case "Extended":
-                        Control1.SelectionMode = ListViewSelectionMode.Extended;
+                        ContentGridView.SelectionMode = ListViewSelectionMode.Extended;
                         break;
                 }
             }
         }
+
+        private void StyledGrid_IncreaseColSpace(object sender, RoutedEventArgs e)
+        {
+            ActualColSpace += 1;
+
+            // Update text box with newly increased value
+            ColSpace.Text = ActualColSpace.ToString();
+            ColSpace.PlaceholderText = ActualColSpace.ToString();
+
+            // Increase left/right margin on all GridViewItems
+            for (int i = 0; i < StyledGrid.Items.Count; i++)
+            {
+                GridViewItem item = StyledGrid.ContainerFromIndex(i) as GridViewItem;
+
+                // Create a Thickness property to bind to each item's Margin property.
+                // Use existing margin and only change relevant values (left/right)
+                Thickness NewMargin = item.Margin;
+                NewMargin.Left = item.Margin.Left + 1;
+                NewMargin.Right = item.Margin.Right + 1;
+
+                item.Margin = NewMargin;
+            }
+
+            NotifyPropertyChanged();
+        }
+
+        private void StyledGrid_DecreaseColSpace(object sender, RoutedEventArgs e)
+        {
+            // Make sure new value is positive
+            if (ActualColSpace - 1 >= 0)
```

Comment:
We can simply check if ActualColSpace is larger than 0, which is a bit easier to read in my opinion.